fb-remote.json

```diff
@@ -0,0 +1,4 @@
+{
```

Comment:
Why is this a json file? Since it is committed, might as well be in the TS file that includes it

Reply:
If anything, this can go into env.json or appConfig.ts so we can configure it per whitelabel app